changelogs/fragments/6712.yml

```diff
@@ -0,0 +1,2 @@
+feat:
+- Add config for topRightNavigation ([#6712](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6712))
```

Comment:
nit i would update the pr changelog to include the actual configuration and specify the dev tool top right nav button

Reply:
Thank you, updated.